README

```diff
@@ -66,6 +66,36 @@ In addition to the usual options, configure understands the following:
        this value is actually supplied as "\\\\".
 
 
+Building on macOS
+-----------------
+
+The following instructions apply to both Apple Silicon and Intel Macs.
+
+To build plptools on macOS, you need to install the GNU autotools and the
+GNU readline library (macOS integrates a different readline version from
+the BSD libedit library).
+Considering the command line tools are already installed and Homebrew is
+your package manager:
+
+brew install autoconf automake libtool pkg-config gettext
+brew install readline
+
+If building from a git checkout, after running 'autoreconf -i', you have
```

Comment:
This doesn't sound like it should be a platform-specific problem. What goes wrong for you? I just tried with a fresh checkout of your branch on GNU/Linux, and it built with no problem.

Reply:
If you don’t install GNU readline, the build will use macOS’ BSD libedit readline without history support. Worse, it misses the `rl_filename_quoting_desired` variable which makes build fail. Didn't you encounter this error when compiling?

Reply:
Sorry, it wasn't clear what I was referring to here. I meant the bit "you have to delete the `intl/VERSION` file".

Reply:
Ah, OK! Do you have Xcode or Apple Command Line Tools installed on your system? The macOS SDK contains header files with `#include "VERSION"`… and `intl/VERSION` is the first one found by the compiler (and it’s not even a header file).

Reply:
I have a macOS VM _in extremis_ but don't normally use macOS! I see the problem. I think the system header files should be using a system include path by default, and we're not messing with the system include path but whatever. So it makes sense that it is a macOS-only issue, and maybe you could mention that the file is deleted so it doesn't shadow a system include file?